src/Native/GirTestLib/girtest-typed-record-tester.c

```diff
@@ -224,6 +224,18 @@ int girtest_typed_record_tester_get_ref_count_sum_nullable(GirTestTypedRecordTes
     return girtest_typed_record_tester_get_ref_count_sum(data, size);
 }
 
+/**
+ * girtest_typed_record_tester_out_no_ownership_transfer:
```

Comment:
Perhaps name the method: `girtest_typed_record_tester_out_no_ownership_transfer_non_nullable_caller_allocates`?